.editorconfig

```diff
@@ -0,0 +1,10 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*.c]
```

Comment:
This is what I had in another branch: https://github.com/hexchat/hexchat/blob/wip/meson/.editorconfig